modules/file-rotation/file-rotation.h

```diff
@@ -42,22 +41,16 @@ struct _FileRotationPlugin
   LogDriverPlugin super;
   SignalSlotConnector *ssc;
   gchar *interval;
+  gchar *date_format;
   gsize size;
   gchar *filename;
-};
-
-struct _FileRotationResponseData
-{
-  FileRotationResult result;
-  GString *filename;
-  gint64 last_rotation;
-  gint64 last_size_check;
-  gint64 last_interval_check;
+  time_t *last_rotation_time;
```

Comment:
Are you planning to use this field for something?

Reply:
Thought of using this field to keep a track of last rotated time